src/Features/CSharp/Portable/Completion/KeywordRecommenders/IntoKeywordRecommender.cs

```diff
@@ -40,6 +40,17 @@ private bool IsValidContextForSelect(CSharpSyntaxContext context)
 
             var lastToken = select.Expression.GetLastToken(includeSkipped: true);
 
+            var memberAccessExpression = select.Expression as MemberAccessExpressionSyntax;
+            if(memberAccessExpression  != null && string.IsNullOrEmpty(memberAccessExpression.Name.ToString()))
+            {
+                return false;
+            }
+
+            //if(lastToken.Kind() == SyntaxKind.DotToken)
```

Comment:
@CyrusNajmabadi is referring to [this extension method](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Extensions/SyntaxTokenExtensions.cs,1e76181b2b7fe49d).